ui/app/components/app/transaction-list-item/transaction-list-item.component.js

```diff
@@ -22,6 +22,8 @@ import {
 import { useShouldShowSpeedUp } from '../../../hooks/useShouldShowSpeedUp'
 import TransactionStatus from '../transaction-status/transaction-status.component'
 import TransactionIcon from '../transaction-icon'
+import { useTransactionTimeRemaining } from '../../../hooks/useTransactionTimeRemaining'
+import IconWithLabel from '../../ui/icon-with-label/icon-with-label'
```

Comment:
I noticed that you did add an `index.js` file for `icon-with-label`, but the file appears to be empty :thinking:

Reply:
oops

Reply:
resolved